sim/dex-moves.ts

```diff
@@ -326,6 +326,8 @@ export interface ActiveMove extends MutableMove, RuinableMove {
 	typeChangerBoosted?: Effect;
 	willChangeForme?: boolean;
 	infiltrates?: boolean;
+	/** Should Order Up try its after-move boost effect?*/
+	orderUpBoost?: boolean;
```

Comment:
why is this here

Reply:
No idea where this is, as dex-moves doesn't show on the current list of changed files. Resolving for now.